<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Forensic Analizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutLeft">
      <item>
       <widget class="QToolBox" name="itemToolBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="videoPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>516</width>
           <height>192</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Videos</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="videoList">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="eventPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>516</width>
           <height>192</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Events</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="eventList">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="individualPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>516</width>
           <height>192</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Individuals</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="indiviualList">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="propertiesForm" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutRight">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="VideoPlayer" name="player" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="rewindButton">
           <property name="text">
            <string>Rewind</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stepButton">
           <property name="text">
            <string>Step</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>&amp;Video</string>
    </property>
    <addaction name="actionAdd_Video_File"/>
    <addaction name="action_Remove_From_Project"/>
   </widget>
   <widget class="QMenu" name="menuEvent">
    <property name="title">
     <string>&amp;Event</string>
    </property>
    <addaction name="actionNewEvent"/>
    <addaction name="actionDeleteEvent"/>
    <addaction name="separator"/>
    <addaction name="actionAuto_Detect"/>
   </widget>
   <widget class="QMenu" name="menuIndividual">
    <property name="title">
     <string>&amp;Individual</string>
    </property>
    <addaction name="actionNewIndividual"/>
    <addaction name="actionDeleteIndividual"/>
    <addaction name="separator"/>
    <addaction name="actionAuto_Detect_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
   <addaction name="menuEvent"/>
   <addaction name="menuIndividual"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="projectToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_Project"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNewEvent">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionDeleteEvent">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionNewIndividual">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionDeleteIndividual">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionAdd_Video_File">
   <property name="text">
    <string>&amp;Add Video File</string>
   </property>
  </action>
  <action name="action_Remove_From_Project">
   <property name="text">
    <string>&amp;Remove From Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="text">
    <string>Save Project As</string>
   </property>
  </action>
  <action name="actionAuto_Detect">
   <property name="text">
    <string>Auto Detect</string>
   </property>
  </action>
  <action name="actionAuto_Detect_2">
   <property name="text">
    <string>Auto Detect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoPlayer</class>
   <extends>QWidget</extends>
   <header>videoplayer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
